lib/zclient.h

```diff
@@ -245,7 +246,8 @@ struct zapi_ipv6
 };
 
 extern int zapi_ipv6_route (u_char cmd, struct zclient *zclient, 
-                     struct prefix_ipv6 *p, struct zapi_ipv6 *api);
+                     struct prefix_ipv6 *p, struct prefix_ipv6 *src_p,
+                     struct zapi_ipv6 *api);
```

Comment:
Then why do we need to have a flag in zapi_ipv6 to tell the receiver function it's being used?